src/App.js

```diff
@@ -1,21 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import { AddItemView } from './components/AddItemView';
 import { ListView } from './components/ListView';
+import Home from './components/Home';
 import { RouteLink } from './components/RouteLink';
 import './App.css';
-import List from './List';
 
 function App() {
+  const [token, setToken] = useState('');
```

Comment:
Thanks again for thinking ahead and moving this state up to App.js! Made our hand-off super easy.

Reply:
Agreed! Good idea 👏🏽

Reply:
Thanks for finding the edge case, Yenly. I think checking if a token exists before sending it to the db, is a good approach.